scripts/update_birdnet_snippets.sh

```diff
@@ -14,6 +14,11 @@ if [ -f $my_dir/privacy_server ] || [ -L /usr/local/bin/privacy_server.py ];then
   rm -f $my_dir/privacy_server.py
   rm -f /usr/local/bin/privacy_server.py
 fi
+
+# Adds python virtual-env to the python systemd services
+sudo -E sed -i "s|ExecStart=.*|ExecStart=$HOME/BirdNET-Pi/birdnet/bin/python3 /usr/local/bin/server.py|" ~/BirdNET-Pi/templates/birdnet_server.service
```

Comment:
Woah!! I didn't know `sed` could use the pipe as a delimiter like that! Super helpful for readability when paths are involved. :+1:

Reply:
Yeah! I learned this because `$HOME` contained an un-escaped `/` and it was giving sed some issues.